lib/webauthn/encoder.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module WebAuthn
+  class Encoder
+    attr_reader :encoding
+
+    def initialize(encoding = :base64)
+      @encoding = encoding
+    end
+
+    def encode(data)
+      case encoding
+      when :base64
+        Base64.strict_encode64(data)
+      when :base64url
+        Base64.urlsafe_encode64(data, padding: false)
+      when nil
```

Comment:
Change to `when nil, false` so that users can say `encoding: false` which reads better than `encoding: nil`.